sdk/react/bottomOverlay.js

```diff
@@ -149,8 +149,8 @@ class BottomOverlay extends Component {
         testID={VIEW_NAMES.TIME_SEEK_BAR_THUMB}
         accessible={false}
         importantForAccessibility='no-hide-descendants'
-        accessibilityLabel={VIEW_NAMES.TIME_SEEK_BAR_THUMB}
-        style={[scrubberStyle, positionStyle, { width: scrubberSize, height: scrubberSize }]}>
+        accessibilityLabel=''
+        style={[scrubberStyle, positionStyle, {width:scrubberSize, height:scrubberSize}]}>
```

Comment:
Left option is preferred according to code style introduced by linter.

Reply:
done